commit 02d8f075ac44f6f0dc46881965f815576921f2c0
Author: Po-Hsu Lin <po-hsu.lin@canonical.com>
Date:   Tue Jun 27 14:17:14 2017 +0800

    selftests: add missing test name in memory-hotplug test
    
    There is no prompt for testing memory notifier error injection,
    added with the same echo format of other tests above.
    
    Signed-off-by: Po-Hsu Lin <po-hsu.lin@canonical.com>
    Signed-off-by: Shuah Khan <shuahkh@osg.samsung.com>

diff --git a/tools/testing/selftests/memory-hotplug/mem-on-off-test.sh b/tools/testing/selftests/memory-hotplug/mem-on-off-test.sh
index c735ece968f8..0f62c16c35c4 100755
--- a/tools/testing/selftests/memory-hotplug/mem-on-off-test.sh
+++ b/tools/testing/selftests/memory-hotplug/mem-on-off-test.sh
@@ -207,6 +207,7 @@ prerequisite_extra()
 	fi
 }
 
+echo -e "\t Test with memory notifier error injection"
 prerequisite_extra
 
 #